content/en/continuous_integration/static_analysis/_index.md

```diff
@@ -95,26 +94,33 @@ Configure the following environment variables:
 | `DD_APP_KEY` | Your Datadog application key. This key is created by your [Datadog organization][102] and should be stored as a secret.    | Yes      |                 |
 | `DD_SITE`    | The [Datadog site][103] to send information to. Your Datadog site is {{< region-param key="dd_site" code="true" >}}.       | No       | `datadoghq.com` |
 
-Provide the following inputs:
+| Name        | Description                                                                                                                | Required | Default         |
+|-------------|----------------------------------------------------------------------------------------------------------------------------|----------|-----------------|
+| `service`   | The name of the service to tag the results with.                                                                           | Yes      |                 |
+| `env`       | The environment to tag the results with. `ci` is a helpful value for this input.                                           | No       | `none`          |
+| `cpu_count` | Set the number of CPUs used by the analyzer. Defaults to the number of CPUs available.                                      | No       |                 |
 
-| Name       | Description                                                                                                                | Required | Default         |
-|------------|----------------------------------------------------------------------------------------------------------------------------|----------|-----------------|
-| `service`  | The name of the service to tag the results with.                                                                           | Yes      |                 |
-| `env`      | The environment to tag the results with. `ci` is a helpful value for this input.                                           | No       | `none`          |
+<div class="alert alert-info">
+  Add a `--performance-statistics` flag to your static analysis command to get execution time statistics for analyzed files.
+</div>
 
 Add the following to your CI pipeline:
```

Comment:
We should add a note that the following script works for `x86_64` on Linux. We should list all the architecture available and their link to to the latest download.